gce_image_import_export_tests/test_suites/import/image_import_test_suite.go

```diff
@@ -99,6 +102,25 @@ func runImageImportOSFromImageTest(
 	verifyImportedImage(ctx, testCase, testProjectConfig, imageName, logger)
 }
 
+// Test most of params except -oauth, -compute_endpoint_override, and -scratch_bucket_gcs_path
+func runImageImportWithRichParamsTest(
+		ctx context.Context, testCase *junitxml.TestCase,
+		logger *log.Logger, testProjectConfig *testconfig.Project) {
+
+	suffix := pathutils.RandString(5)
+	imageName := "e2e-test-image-import-data-disk-" + suffix
+	cmd := "gce_vm_image_import"
+	args := []string{"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+		fmt.Sprintf("-image_name=%s", imageName), "-data_disk", fmt.Sprintf("-source_file=gs://%v-test-image/image-file-10g-vmdk", testProjectConfig.TestProjectID),
+		"-no_guest_environment", "-family=test-family", "-description=test-description",
```

Comment:
Should be easy to verify: family, description and labels, as those are image properties.